massa_test_framework/compile.py

```diff
@@ -26,7 +26,7 @@ def apply(self, root=Path, strip: int = 0, fuzz: bool = False):
         with open(root / self.constant_file, "r+") as fp:
             content = fp.read()
             content_sub = re.sub(
-                f"(pub )?const {self.constant_name}: (\w+) = (\w+);",
+                f"(pub )?const {self.constant_name}: ([\w\<\>]+) = ([\w\s\*\(\)]+);",
                 f"\g<1>const {self.constant_name}: \g<2> = {self.new_value};",
                 content,
             )
```

Comment:
```suggestion
		content_sub = re.sub(
			f"(.* const {self.constant_name}[ :].*) = (.*);",
			f"\g<1> = {self.new_value};",
            content,
        )
```
looks more readable to me
we capture the right member of the `=` sign as long as it contains the correct identifier
then we replace the whole line by `<1> = new_value;`